Antidote/AvatarsManager.m

```diff
@@ -98,6 +98,44 @@ - (UIImage *)createAvatarFromString:(NSString *)string diameter:(CGFloat)diamete
     return [self imageWithView:label];
 }
 
+- (UIImage *)createAvatarFromString:(NSString *)string diameter:(CGFloat)diameter
+                          textColor:(UIColor *)textColor backgroundColor:(UIColor *)backgroundColor
```

Comment:
I am not sure what you mean? There is already one using the color from `appearance`. However I am trying to get the white instead of the app's tint color. 

Ohh, I think you mean using the other colors available from the `AppearanceManager`, like `- (UIColor *)textMainColor;`?